website/docs/modules/ROOT/partials/Publishing_Footer.adoc

```diff
@@ -1,4 +1,57 @@
-== Publishing to Sonatype Maven Central
+== `SonatypeCentralPublishModule` Configurations
+
+
+Below you are values you could override in the `SonatypeCentralPublishModule`.
+
+=== Module Settings
```

Comment:
This should be moved to the bottom of this section, since most people should not need to configure them.

You should also mention that  they can be set via command line flags and provide an example